backend/app/routers/groups.py

```diff
@@ -42,30 +33,23 @@ async def save_group(
 
 @router.get("/{group_id}", response_model=GroupOut)
 async def get_group(group_id: str, db: MongoClient = Depends(dependencies.get_db)):
-    # Check all connection and abort if any one of them is not available
-    if db is None:
-        raise HTTPException(status_code=503, detail="Service not available")
-        return
     if (group := await db["groups"].find_one({"_id": ObjectId(group_id)})) is not None:
         return GroupOut.from_mongo(group)
     raise HTTPException(status_code=404, detail=f"Group {group_id} not found")
 
 
-@router.post("/{group_id}", response_model=GroupOut)
+@router.put("", response_model=GroupOut)
```

Comment:
Any reason you leave the `group_id` out of the path parameter?

Reply:
no that was not intentionally removed, fixed thanks